core/src/main/java/org/stellar/anchor/sep10/Sep10Helper.java

```diff
@@ -0,0 +1,38 @@
+package org.stellar.anchor.sep10;
+
+import static org.stellar.anchor.util.Log.debugF;
+import static org.stellar.anchor.util.Log.infoF;
+
+import java.io.IOException;
+import org.stellar.anchor.api.exception.SepException;
+import org.stellar.anchor.util.Sep1Helper;
+import org.stellar.sdk.FormatException;
+import org.stellar.sdk.KeyPair;
+
+public class Sep10Helper {
+  public static String getClientAccountId(String clientDomain) throws SepException {
```

Comment:
This function is complex. Let's start adding comments.

Reply:
How about rename to `fetchSigningKey()`?

Reply:
Yeah this function is moved from Sep10Service for reuse, yeah I agree `fetchSigningKey` would be a more proper name